elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -23,7 +23,8 @@
  */
 public class Util {
 
-    public static EntityManagerFactory getEntityManagerFactory(String modelPackageName, Properties options) {
+    public static EntityManagerFactory getEntityManagerFactory(String modelPackageName, String asyncModelPackageName, 
```

Comment:
Instead of passing the package name for async here (which Elide already knows), you could just pass a flag about whether or not to include async models.

Reply:
Done